opendevin/runtime/client/client.py

```diff
@@ -75,13 +76,23 @@ def _run_command(self, command: str) -> Observation:
             )
         except UnicodeDecodeError:
             return ErrorObservation('Command output could not be decoded as utf-8')
-           
+
+    def clean_up(self,input_text):
+        # Remove escape sequences
+        cleaned_text = re.sub(r'\x1b\[[0-9;?]*[a-zA-Z]', '', input_text)
```

Comment:
Can we extract and reuse `strip_ansi` from `/Users/liboxuan/workspace/OpenDevin/opendevin/runtime/plugins/jupyter/execute_server`?

Aside, I don't understand why this isn't needed in the old ssh sandbox.

Reply:
Discussed offline. The `clean_up` function used when clean some specific escape sequences generated in terminal.